ffs/scheduler/scheduler.go

```diff
@@ -85,9 +85,6 @@ func (s *Scheduler) push(iid ffs.APIID, waddr string, cfg ffs.CidConfig, oldCid
 	if err := cfg.Validate(); err != nil {
 		return ffs.EmptyJobID, fmt.Errorf("validating cid config: %s", err)
 	}
-	if waddr == "" {
-		return ffs.EmptyJobID, fmt.Errorf("invalid wallet address")
-	}
```

Comment:
Yeah, this was mostly a safety net.

Do you mind adding this check now in `ffs/types.go` on `func (cc ColdConfig) Validate()`?

Reply:
👍